src/content/get-started/install/index.mdx

```diff
@@ -23,6 +23,16 @@ If you need help installing Okteto on your Kubernetes cluster, we have a playlis
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/videoseries?list=PLVcXGaCKg-Y0XGujg_G7vCIGd_JEAQ4Sq" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
 
+## Supported Browsers
+
+The Okteto Dashboard is optimized for the following browsers:
+- **Google Chrome**: Latest 2 major versions
+- **Mozilla Firefox**: Latest 2 major versions
+- **Microsoft Edge**: Latest 2 major versions
+- **Safari**: Latest 2 major versions (macOS only)
```

Comment:
@codyjlandstrom Most browsers don't use the classic semantic versioning based on MAJOR.MINOR.PATCH. So that's the reason why we were supponting only the 8 last versions. In the case of Safari, they do use semantic versioning and it makes sense to use the "Last 2 major versions", it actually supports a couple more versions than using just the "last 8 versions" for this browser, so I will adjust it in browserslist.

```suggestion
The Okteto Dashboard is optimized for the following browsers:
- **Google Chrome**: Last 8 versions
- **Mozilla Firefox**: Last 8 versions
- **Microsoft Edge**: Last 8 versions
- **Safari**: Last 2 major versions (macOS only)
```

The bundle size doesn't change if we do that change to Safari only.

Reply:
This is our configuration and our browsers support according to "browserlist":
<img width="835" alt="Screenshot 2025-01-22 at 11 32 06" src="https://github.com/user-attachments/assets/e1d864f6-31dd-4a79-9ccb-ff090d01e43d" />

Reply:
Browserslist changes in this PR: https://github.com/okteto/app/pull/9073